compiler/src/iree/compiler/Dialect/Flow/IR/FlowOps.cpp

```diff
@@ -644,17 +644,17 @@ LogicalResult DispatchTensorLoadOp::reifyResultShapes(
       if (droppedDims.test(mixedSize.index())) {
         continue;
       }
-      shape.push_back(
-          getValueOrCreateConstantIndexOp(b, getLoc(), mixedSize.value()));
+      shape.push_back((Value)getValueOrCreateConstantIndexOp(
+          b, getLoc(), mixedSize.value()));
```

Comment:
I feel that mixedSizes are OpFoldResult types. Can we append it directly?

```suggestion
      shape.push_back(mixedSize.value());
```